defold-spine/commonsrc/vertices.cpp

```diff
@@ -200,33 +224,42 @@ uint32_t CalcVertexBufferSize(const spSkeleton* skeleton, spSkeletonClipping* sk
     {
         spSlot* slot = skeleton->drawOrder[s];
         spAttachment* attachment = slot->attachment;
-
         if (!attachment)
         {
             spSkeletonClipping_clipEnd(skeleton_clipper, slot);
             continue;
         }
 
+        if (slot->color.a == 0 || !slot->bone->active)
+        {
+            spSkeletonClipping_clipEnd(skeleton_clipper, slot);
+            continue;
+        }
+
         if (attachment->type == SP_ATTACHMENT_CLIPPING)
         {
             spClippingAttachment* clip = (spClippingAttachment*) attachment;
             spSkeletonClipping_clipStart(skeleton_clipper, slot, clip);
             continue;
         }
-
-        if (spSkeletonClipping_isClipping(skeleton_clipper))
+        if (attachment->type == SP_ATTACHMENT_REGION || attachment->type == SP_ATTACHMENT_MESH)
```

Comment:
As we do check the type several times, it is likely good to store the type on line 238, and reuse it.
It also saves on pointer dereferencing, which is generally good for performance as well (compiler can assume more about code paths)